.github/workflows/integration-tests.yml

```diff
@@ -109,6 +109,15 @@ jobs:
           # Run hopper/test_flashattention.py separately to avoid out of gpu memory
           python3 -m pytest -vs hopper/test_flashattention.py
 
+      - name: Run python examples on CUDA
+        # Execute on H100 due to its ~30s speed advantage over A100;
+        # example duration matches this gain and is device-agnostic.
+        if: ${{(matrix.runner[0] == 'self-hosted') && (matrix.runner[1] == 'H100')}}
+        run: |
+          cd python/examples
+          # Test example
```

Comment:
Is this comment helpful, since we're under the `Run python examples on CUDA` header?  If it is helpful (maybe it shows up in CI whereas the header doesn't?) then please change to "Test example*s*".